dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionletTest.java

```diff
@@ -83,48 +205,77 @@ private static ContentType createTestType(final ContentTypeAPI contentTypeAPI)
     @AfterClass
     public static void cleanup()
             throws DotDataException, DotSecurityException {
+        for (ContentType contentType : contentTypes) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'contentType' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337055)